src/main/java/appeng/client/gui/implementations/GuiStorageBus.java

```diff
@@ -114,6 +118,10 @@ public void drawFG(final int offsetX, final int offsetY, final int mouseX, final
         if (this.rwMode != null) {
             this.rwMode.set(((ContainerStorageBus) this.cvb).getReadWriteMode());
         }
+
+        if (this.stickyMode != null) {
+            this.stickyMode.set(((ContainerStorageBus) this.cvb).getStickyMode());
```

Comment:
It is better to check it via instanceof to be sure this not going into ClassCastException

Reply:
Should I change the other checks in the file as well to comply with this?

Reply:
You can just check it using 
```java
if (this.cvb inctanceof ContainerStorageBus csb) {
   csb.getStickyMode()
}
```

Reply:
I actually am going to remove this button as the sticky mode isn't controllable through a button, only a card. Forgot about that on accident.

Reply:
I went ahead and changed the other checks in this file to comply with your request.